cyclus/system.py

```diff
@@ -0,0 +1,56 @@
+"""Some system-specific info for cyclus."""
+import sys
+import importlib
+
+from cyclus.lazyasd import lazyobject
+
+
+@lazyobject
+def PY_VERSION_TUPLE():
+    return sys.version_info[:3]
+
+
+@lazyobject
+def curio():
+    if PY_VERSION_TUPLE < (3, 5, 2):
+        return None
+    else:
+        try:
+            return importlib.import_module('curio')
+        except ImportError:
+            return None
```

Comment:
Am I correct in assuming that these are importing parts of the code? Should we return an error message if it can't find these imports?